### PR TITLE
Add psc-package support

### PR DESCRIPTION
**Pull request by lostbean**
This is just a small PR to update the module and add psc-package support.

### CODE REVIEW

File: psc-package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "erl-process",
+    "set":  "erl-0.10.1",
```
Comment (nwolverson) [inline psc-package.json]: Does this work with the latest package set? erl-0.10.2-4

File: Makefile [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-
-test: ps erl
-	erl -pa ebin -noshell -eval '(test_main:main())()' -eval 'init:stop()'
+	psc-package sources | xargs pserlc 'test/**/*.purs' 'src/**/*.purs'
 
 erl:
 	mkdir -p ebin
 	erlc -o ebin/ output/*/*.erl
+
+run:
```
Comment (nwolverson) [inline Makefile]: Not sure why this is renamed, isn't this running some kind of test?

File: src/Erl/Process.purs [hunk 3]
```diff
@@ -10,10 +10,10 @@ module Erl.Process
   , module RawExport
   ) where
 
-import Prelude
+import Prelude (class Eq, Unit, eq, (<$>))
```
Comment (nwolverson) [inline src/Erl/Process.purs]: Rather keep `Prelude` imported implicitly, it is other imports which should be explicitly qualified (in this case `Control.Monad.Eff` should be

**Review by nwolverson — CHANGES_REQUESTED** [review]
Thanks! Hope this is somehow useful to you, interested to hear any thoughts

File: Makefile (hunk 2 above)
Comment (lostbean) [reply Makefile]: no special reason, but I agree it should be `test`

File: psc-package.json (hunk 1 above)
Comment (lostbean) [reply psc-package.json]: Oops! I missed the new version. But it does work.

File: src/Erl/Process.purs (hunk 3 above)
Comment (lostbean) [reply src/Erl/Process.purs]: I'm also rather keep `Prelude` implicit.

Wow, I just understood  how the implicit import warning work in purescript :) Only one import is allowed to be implicit. And, in this case, makes total sense to let `Prelude` to be the one :)

**Review by nwolverson — APPROVED** [review]

**Pull request merged**